fwoa-tools/src/migrationImport.ts

```diff
@@ -73,9 +89,61 @@ async function startImport(folderNames: string[]): Promise<void> {
           .promise();
         if (jobStatus.ImportJobProperties.JobStatus === 'COMPLETED') {
           console.log(`successfully imported folder ${folderName}`);
```

Comment:
I think it would be cleaner to put all logic after the import is completed in a different function

Reply:
Makes sense, will update